product_form_sale_link/tests/test_product.py

```diff
@@ -0,0 +1,75 @@
+from odoo.tests import TransactionCase
+from odoo.tests.common import new_test_user, users
+
+
+class TestProductSalesCount(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.user_salesman = new_test_user(
+            cls.env,
+            login="test_salesman",
+            groups="sales_team.group_sale_manager,sales_team.group_sale_salesman,base.group_user,base.group_partner_manager",
+        )
+        cls.user_employee = new_test_user(
+            cls.env,
+            login="test_employee",
+            groups="base.group_user,product.group_product_manager",
+        )
+        cls.product1 = cls.env["product.product"].create(
+            {
+                "name": "Test Product 1",
+            }
+        )
+        cls.product2 = cls.env["product.product"].create(
+            {
+                "name": "Test Product 2",
+            }
+        )
+        cls.product_template = cls.product1.product_tmpl_id
+        cls.partner = cls.env["res.partner"].create(
+            {
+                "name": "Test Partner",
+            }
+        )
```

Comment:
Remove the trailing comma for not having so many useless lines (the same for all).

```suggestion
        cls.partner = cls.env["res.partner"].create(
            {"name": "Test Partner"}
        )
```